src/pages/api/auth/[...nextauth].js

```diff
@@ -1,6 +1,7 @@
 import NextAuth from 'next-auth';
 import CredentialsProvider from 'next-auth/providers/credentials';
 import GithubProvider from 'next-auth/providers/github';
+import GoogleProvider from 'next-auth/providers/google';
```

Comment:
content from another topic ?  ;)

Reply:
ohhh, I forgot to delete it :D 
Wanted to install google login with next auth. But google wants to have all the data (creditcard, etc.) and  therefore I skipped the task. Many thanks for your feedback Coach J :) 💯